server/graphql/v2/mutation/ExpenseMutations.ts

```diff
@@ -183,7 +183,8 @@ const expenseMutations = {
       // NOTE(oauth-scope): Ok for non-authenticated users, we only check scope
       enforceScope(req, 'expenses');
 
-      const getId = (reference: { id?: string | number; legacyId?: number }) => reference?.id || reference?.legacyId;
+      const getId = (reference: { id?: string | number; legacyId?: number; slug?: string }) =>
```

Comment:
:ice_cream: Explicit variable name

```suggestion
      const isExistingAccountReference = (reference: { id?: string | number; legacyId?: number; slug?: string }) =>
```